### PR TITLE
qbe: init at unstable-2019-05-15

### PR DESCRIPTION
**Pull request by fgaz**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/compilers/qbe/default.nix [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    rev = "acc3af47330fd6610cf0fbdb28e9fbd05160888f";
+    sha256 = "1c8ynqbakgz3hfdcyhwdmz7i1hnyd9m25f9y47sc21bvxwfrbzpi";
+  };
+
+  makeFlags = [ "PREFIX=$(out)" ];
+  meta = with stdenv.lib; {
+    homepage = "https://c9x.me/compile/";
+    description = "A small compiler backend written in C";
+    maintainers = with maintainers; [ fgaz ];
+    license = licenses.mit;
```
Comment (Infinisil) [inline pkgs/development/compilers/qbe/default.nix]: `platforms` is missing

Comment (fgaz) [reply pkgs/development/compilers/qbe/default.nix]: From the website:
>  QBE is known to compile and pass its test suite on the following x64 operating systems.
>
>    Linux
>    FreeBSD
>    Mac OS X 

...but the aarch64 builder managed to build it, so I guess "all" is appropriate. Can you trigger a darwin build?

**Comment by Infinisil** [comment]
@GrahamcOfBorg build qbe

Unless we find one that doesn't work, `all` seems fine

File: pkgs/development/compilers/qbe/default.nix (hunk 1 above)
Comment (fgaz) [reply pkgs/development/compilers/qbe/default.nix]: seems to work on darwin

**Comment by fgaz** [comment]
Ping?

**Review by infinisil — APPROVED** [review]

**Pull request merged**